AdyenDropIn/Presentation/DropInNavigationController.swift

```diff
@@ -114,18 +116,14 @@ extension DropInNavigationController: UINavigationControllerDelegate {
 }
 
 extension DropInNavigationController: UIViewControllerTransitioningDelegate {
-    
-    /// :nodoc:
-    public func presentationController(forPresented presented: UIViewController,
-                                       presenting: UIViewController?,
-                                       source: UIViewController) -> UIPresentationController? {
+
+    internal func presentationController(forPresented presented: UIViewController,
+                                         presenting: UIViewController?,
+                                         source: UIViewController) -> UIPresentationController? {
```

Comment:
can we make this method return a non-optional value? same at line #109

Reply:
How do you mean?
Those methods are in UIKit: `UIViewControllerTransitioningDelegate` and `UINavigationControllerDelegate`.

Reply:
was not aware of that 😅